README.md

```diff
@@ -73,7 +75,7 @@ The Ginkgo DPC++(SYCL) module has the following __additional__ requirements:
 
 The Ginkgo MPI module has the following __additional__ requirements:
 
-* MPI 3.1+, ideally with GPUDirect support for best performance
+* MPI 3.1+, ideally GPU-Aware, for best performance
```

Comment:
I think GPU-Aware is different from GPUDirect.
GPU-Aware only recognizes whether the memory is on GPU such that we do not need to move the data to host.
GPU direct is more relevant for performance

Reply:
No, GPUDirect is a NVIDIA related terminology that is unrelated to MPI. The correct term for MPI implementation is GPU-Aware is used across all MPI implementations. See for example [paper from the MVAPICH2 team](https://ieeexplore.ieee.org/document/10171511/)

Reply:
GPU-Aware means that the MPI implementation is able to accept GPU pointers, it does not say whether they actually use direct connections between GPUs. That is what GPUDirect refers to on the NVIDIA side, I'm not sure if there is an equivalent for AMD or Intel.